website/docs/r/efs_file_system.html.markdown

```diff
@@ -22,6 +24,18 @@ resource "aws_efs_file_system" "foo" {
 }
 ```
 
+### Using lifecycle policy
+
+```hcl
+resource "aws_efs_file_system" "foo_with_lifecyle_policy" {
+  creation_token = "my-product"
+
+  lifecycle_policy = {
```

Comment:
This new attribute is a configuration block (TypeList + Elem: Resource), so the equals sign must be omitted in Terraform 0.12 and later.

```suggestion
  lifecycle_policy {
```